### PR TITLE
 Fix font style for the description label on the RA Onboarding screen

### PR DESCRIPTION
**Pull request by zladzeyka**
Fix font style for the description label on the RA Onboarding screen

Figma to check: https://www.figma.com/file/mikn8kCBGSIil4DhDg7NNL/iOS-Gini-Bank-SDK-3.8-UI-Customisation?type=design&node-id=0%3A1&mode=design&t=6RBSLUCqtJ3nLUS1-1

PP-468
Simulator Screenshot - iPhone 15 Plus - 2024-05-16 at 16 20 39

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift [hunk 1]
```diff
@@ -23,7 +23,7 @@ final class DigitalInvoiceCoordinator: Coordinator {
 
     // TODO: This is to cope with the screen coordinator being inadequate at this point to support the return assistant step and needing a refactor.
     // Remove ASAP
-    private var analysisDelegate: AnalysisDelegate
+    weak private var analysisDelegate: AnalysisDelegate?
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift:26]: [suggestions]: Can we order like this: `private weak var` ? 
Starting with the access control modifier is generally considered a good practice for readability and consistency.

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift [hunk 2]
```diff
@@ -23,7 +23,7 @@ final class DigitalInvoiceCoordinator: Coordinator {
 
     // TODO: This is to cope with the screen coordinator being inadequate at this point to support the return assistant step and needing a refactor.
     // Remove ASAP
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift:25]: [clarification]: this comment is still needed? `// Remove ASAP`

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift [hunk 3]
```diff
@@ -81,7 +81,10 @@ extension DigitalInvoiceCoordinator: DigitalInvoiceViewModelDelagate {
     }
 
     func didTapPay(on viewModel: DigitalInvoiceViewModel) {
-        delegate?.didFinishAnalysis(self, invoice: viewModel.invoice, analysisDelegate: analysisDelegate)
+        if let analysisDelegate = analysisDelegate {
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift:84]: [clarification]: what happens if `analysisDelegate` is nil ? Do we need to handle the scenario?

**Review by ValentinaIancu-Gini — COMMENTED** [review]
@zladzeyka Thank you for fixing this.

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift (hunk 3 above)
Comment (zladzeyka) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift:84]: we didn't handle that case in the past - nothing will happened if delegate is nil

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift (hunk 1 above)
Comment (zladzeyka) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift:26]: I've used the automatic function...looks like it's not following the convention :)

**Pull request merged**